community.md

```diff
@@ -0,0 +1,7 @@
+# Community Web Integrations
```

Comment:
Let's include a disclaimer here:

The following projects are not maintained, monitored, moderated, or affiliated with NZXT. Any usage of these projects is at your own risk and NZXT can not be held liable for any of the content or results of running these projects on your NZXT devices.